src/render/filters.rs

```diff
@@ -319,6 +322,75 @@ impl ResolveFilter for SlugifyFilter {
     }
 }
 
+impl ResolveFilter for YesnoFilter {
+    fn resolve<'t, 'py>(
+        &self,
+        variable: Option<Content<'t, 'py>>,
+        py: Python<'py>,
+        template: TemplateString<'t>,
+        context: &mut Context,
+    ) -> ResolveResult<'t, 'py> {
+        let left = match variable {
+            Some(var) => var,
+            None => return Err(PyRenderError::RenderError(RenderError::VariableDoesNotExist {
+                key: "yesno".to_string(),
+                object: "filter".to_string(),
+                key_at: (0, 0).into(),
+                object_at: None,
+            })),
+        };
+        
+        // Handle the case with a custom mapping argument
+        if let Some(arg) = &self.argument {
+            // Get mapping string from the argument
+            let arg_content = arg.resolve(py, template, context)?
+                .expect("missing argument in context should already have raised");
+            let mapping = match arg_content.to_py(py)?.extract::<String>() {
+                Ok(s) => s,
+                Err(_) => {
+                    return Err(PyRenderError::RenderError(RenderError::VariableDoesNotExist {
+                        key: "yesno argument".to_string(),
+                        object: "string".to_string(),
+                        key_at: (0, 0).into(),
+                        object_at: None,
+                    }));
+                }
+            };
+
+            let parts: Vec<&str> = mapping.split(',').collect();
+            
+            // Handle None values
+            if left.to_py(py)?.is_none() {
```

Comment:
I wonder if we should add a new method to `Content` returning an enum with three variants: `None`, `Truthy` and `Falsey`, which we can then match on here.